boards/st/sensortile_box_pro/Kconfig

```diff
@@ -0,0 +1,12 @@
+# SENSORTILE_BOX_PRO board configuration
+
+# Copyright (c) 2024 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_SENSORTILE_BOX_PRO
+
+config BOARD_SERIAL_BACKEND_CDC_ACM
+	bool "Use USB CDC as serial console backend"
+	default y
+
+endif # BOARD_SENSORTILE_BOX_PRO
```

Comment:
maybe you can alternatively put "depends on BOARD_SENSORTILE_BOX_PRO"?

Reply:
Probably, but if further custom config added it will be better to simply add it between if/endif.
And I simply do the same than boards providing this kind of config.